docs/docsite/rst/porting_guides/porting_guide_2.9.rst

```diff
@@ -266,6 +266,9 @@ Noteworthy module changes
 * `vmware_datastore_cluster <vmware_datastore_cluster_module>` accepts `folder` parameter to place datastore cluster in user defined folder. This option makes `datacenter` as an optional parameter.
 * `mysql_db <mysql_db_module>` returns new `db_list` parameter in addition to `db` parameter. This `db_list` parameter refers to list of database names. `db` parameter will be deprecated in version `2.13`.
 * `snow_record <snow_record_module>` and `snow_record_find <snow_record_find_module>` now takes environment variables for `instance`, `username` and `password` parameters. This change marks these parameters as optional.
+* `tower_workflow_template <tower_workflow_template_module>` accepts `ask_inventory` and `inventory` as optional parameters.
```

Comment:
Did this actually happen in 2.9 (since it's already branched) or does this change need to go into `porting_guide_2.10.rst`?

Reply:
Thanks a lot for reviewing. Moved to 2.9 porting guide.